source/_components/android_ip_webcam.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Android IP Webcam"
+description: "Connect Android devices as IP webcam to home-assistant"
+date: 2017-03-10 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: android_ip_webcam.png
+ha_category: Hub
+ha_release: 0.40
+---
+
+The [Android IP webcam](https://play.google.com/store/apps/details?id=com.pas.webcam) turns your andorid phone into a network camera with multiple viewing options.
+
+It setup a mjpeg camera view and all settings as switch inside Home Assistant. You can also expose the sensors. If you have multible device, you can use all options inside a list.
```

Comment:
Suggest

`It's setup as a mjpeg camera and all settings as switches inside of Home Assistant. You can also expose the sensors. If you have multiple devices, you can use all options inside a list.`

Unsure if it's supposed to be `multiple devices` or `multiple sensors`.